exist-parent/pom.xml

```diff
@@ -693,10 +694,8 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
-                    <version>3.13.0</version>
+                    <version>3.14.0</version>
```

Comment:
I should und this, I need help for this @line-o @reinhapa

Reply:
What is the problem with this?

Reply:
it should be part of dependabot;
I was 24hours too early, thought dependabot missed this one :-)
please leave it in